docs/Config_Reference.md

```diff
@@ -318,6 +317,83 @@ radius:
 #   just prior to starting a probe operation. The default is 5.
 ```
 
+### Deltesian Kinematics
+
+See [example-deltesian.cfg](../config/example-deltesian.cfg) for an
+example linear delta kinematics config file. See the
+[delta calibrate guide](Delta_Calibrate.md) for information on
+calibration.
+
+Only parameters specific to deltesian printers are described here - see
+[common kinematic settings](#common-kinematic-settings) for available
+ parameters.
+
+```
+[printer]
+kinematics: deltesian
+max_z_velocity:
+#   For deltesian printers, this limits the maximum velocity (in mm/s) of
+#   moves with z axis movement. This setting can be used to reduce the
+#   maximum speed of up/down moves (which require a higher step rate
+#   than other moves on a deltesian printer). The default is to use
+#   max_velocity for max_z_velocity.
+#max_z_accel:
+#   This sets the maximum acceleration (in mm/s^2) of movement along
+#   the z axis. Setting this may be useful if the printer can reach higher
+#   acceleration on XY moves than Z moves (eg, when using input shaper).
+#   The default is to use max_accel for max_z_accel.
+#minimum_z_position: 0
+#   The minimum Z position that the user may command the head to move
+#   to. The default is 0.
+#min_angle: 5
+#   This represents the minimum angle (in degrees) relative to horizontal
+#   that the deltesian arms are allowed to achieve. This parameter is
+#   intended to restrict the arms from becomming completely horizontal,
+#   which would risk accidental inversion of the XZ axis. The default is 5.
+#print_width:
+#   The distance (in mm) of valid toolhead X coordinates. One may use
+#   this setting to customize the range checking of toolhead moves. If
+#   a large value is specified here then it may be possible to command
+#   the toolhead into a collision with a tower. This setting usually
+#   corresponds to bed width (in mm).
+#slow_ratio: 3
+#   The ratio used to limit velocity and acceleration on moves near the
+#   extremes of the X axis. If vertical distance divided by horizontal
+#   distance exceeds the value of slow_ratio, then velocity and
+#   acceleration are limited to half their nominal values. If vertical
+#   distance divided by horizontal distance exceeds twice the value of
+#   the slow_ratio, then velocity and acceleration are limited to one
+#   quarter of their nominal values. The default is 3.
+
+# The stepper_left section is used to describe the stepper controlling
+# the left tower. This section also controls the homing parameters
+# (homing_speed, homing_retract_dist) for all towers.
+[stepper_left]
```

Comment:
I don't have a strong opinion on this either, though I will note that the lowercase `l` character is ambiguous in many typefaces, and easily confused with an uppercase `I` and/or the number `1`

Using capital `L` and `R`, or writing out `left` and `right`, resolves that ambiguity. Food for thought.

Reply:
Yeah, I'm not sure if Klipper supports uppercase letters in section names. From that point of view, `left` and `right` might be better. Then, the confusion with 'I' or '1' is probably not very likely, especially considering the counterpart `stepper_r`. I just brought this up for discussion since in other cases Klipper uses notion like `stepper_x`(`_y`, `_z`), `stepper_a`(`_b`,`_c`) and such. Though polar kinematics already uses `stepper_arm`, so there's already a similar example in use.

Reply:
As Leapo said `stepper_l` can easily be confused with `stepper_1`. These 3 extra letters make the configuration obvious for the users without even having to check the documentation. Obviously, `stepper_right` must follow the same syntax. My guess is that Klipper also uses the full name of the steppers in the other kinematics; it just leads to one letter in most cases. I would like to stick to `_left`, `_right`.

Reply:
OK, then let's stick with _left and _right notation.